### PR TITLE
file added

### PR DESCRIPTION
**Pull request by saidhanya18**
## Related Issue

I have added windows guidelines for the user

### CODE REVIEW

File: Window_setup.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+To run dynamic cli on windows:
+follow the steps!!
+1. first you will require WSL to run dynamic-cli on windows.
+2. https://docs.microsoft.com/en-us/windows/wsl/install
```
Comment (GouravSardana) [inline Window_setup.txt:4]: Can you use hyperlink and add those links there. One more thing link this file in the readme where we are giving instructions for windows user

**Review by GouravSardana — CHANGES_REQUESTED** [review]
Please make the suggested changes

File: Window_setup.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+3. after that instal python3 using pip ($ sudo apt-get install python3-pip)
```
Comment (pranavbaburaj) [inline Window_setup.txt:5]: typos 🤖

**Review by lainq — CHANGES_REQUESTED** [review]

File: Window_setup.txt (hunk 2 above)
Comment (saidhanya18) [reply Window_setup.txt]: issue solved

**Review by lainq — CHANGES_REQUESTED** [review]
- Rename `Window_setup` to something like `windows_setup`.
- Use markdown file instead of text file, and use proper markdown elements like codeblocks and links.

**Comment by GouravSardana** [comment]
cc @pranavbaburaj @PradyumnaKrishna @peb-peb

File: Window_setup.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+To run dynamic cli on windows:
+follow the steps!!
+1. first you will require WSL to run dynamic-cli on windows.
+2. https://docs.microsoft.com/en-us/windows/wsl/install (hyperlink for WSL)
+3. after that install python3 using pip ($ sudo apt-get install python3-pip)
```
[the remark this answers is not in the document]
Comment (GouravSardana) [reply Window_setup.txt:5]: What if we don’t have python3-pip

**Comment by peb-peb** [comment]
I think that there should no separate Documentation for setting WSL because after installing and setting up WSL, the procedure/steps to setup `dynamic-cli` in it is basically the same as linux. So, just referring to the WSL documentation should be enough in this case.

But, when `dynamic-cli` is available on WINDOWS, then maybe we could have it's separate _documentation_.

**Comment by pranavbaburaj** [comment]
> I think that there should no **separate** Documentation for setting WSL because after installing and setting up WSL, the procedure/steps to setup `dynamic-cli` in it is basically the same as linux. So, just referring to the WSL documentation should be enough in this case.
> 
> But, when `dynamic-cli` is available on **WINDOWS**, then maybe we could have it's separate _documentation_.

hmm yeah, that sounds reasonable.

**Comment by peb-peb** [comment]
So, if everyone agrees with it, then maybe we could close this PR.

@saidhanya18 I'll create a issue regarding this, and then you could follow from there.

**Comment by pranavbaburaj** [comment]
Yes, I think we can also hear from @GouravSardana and @saidhanya18 before closing this

**Comment by GouravSardana** [comment]
> no **separate** Documentation for setting

Sounds good but We need at least a document where we'll give the steps to install WSL

**Comment by peb-peb** [comment]
> We need at least a document where we'll give the steps to install WSL

We could do that, so that new contributors are _not_ confused and could follow the commands directly. But the WSL installation documentation is very detailed in itself and most of the other open-source projects direct their contributors to their site.

So, I think it's a matter of preference. @GouravSardana @pranavbaburaj I would like to know your opinions on this.

**Comment by saidhanya18** [comment]
> So, if everyone agrees with it, then maybe we could close this PR.
> 
> @saidhanya18 I'll create a issue regarding this, and then you could follow from there.

sounds good !

**Comment by GouravSardana** [comment]
Merging the PR. Adding the change now.

**Pull request merged**